### PR TITLE
gg-27070

### PR DESCRIPTION
**Pull request by vmalin**
pull request for review

### CODE REVIEW

**Review by vmalin — COMMENTED** [review]
General comments:
1. Message with cluster name should be same for all commands and should be printed at the same place. For example right after client connection establishing.
2. Not all commands are covered in this branch. For example validate_indexes command wasn't changed.
3. If you make commits via IDEA you can set up optimizing imports before commit so you won't have to think about unused imports

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/ClusterChangeTagCommand.java [hunk 1]
```diff
@@ -66,7 +66,7 @@
             );
 
             if (res.success())
-                logger.info("Cluster tag updated successfully, old tag was: " + res.tag());
+                logger.info( getFullClusterInfo(client.state()) + " tag updated successfully, old tag was: " + res.tag());
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/ClusterChangeTagCommand.java:69]: Excess space in info call.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/Command.java [hunk 2]
```diff
@@ -51,6 +53,29 @@ public static GridClient startClient(GridClientConfiguration clientCfg) throws E
         return client;
     }
 
+    /**
+     * @return Cluster information to show user for.
+     */
+    default String getFullClusterInfo(GridClientClusterState clientCfg){
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/Command.java:59]: This method doesn't actually return full cluster info. Method name should be changed

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/Command.java [hunk 3]
```diff
@@ -51,6 +53,29 @@ public static GridClient startClient(GridClientConfiguration clientCfg) throws E
...
+        String clusterInfo = "Cluster";
+        if (clusterName != null)
+            clusterInfo = "Cluster \"" + clusterName + "\"";
+        return clusterInfo;
+    }
+
+    /**
+     * @return Cluster information to show user for.
+     */
+    default String getClusterInfo(GridClientClusterState clientCfg){
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/Command.java:70]: getClusterInfo -> getClusterName

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/Command.java [hunk 4]
```diff
@@ -51,6 +53,29 @@ public static GridClient startClient(GridClientConfiguration clientCfg) throws E
...
+        String clusterName = null;
+        try{
+            clusterName = clientCfg.clusterName();
+        } catch (GridClientException ignored){
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/Command.java:74]: Try-catch block doesn't comply with our code style. Also there should be some more blank lines. Check out this link: https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines

File: modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java [hunk 5]
```diff
@@ -1673,6 +1674,7 @@ public void testErrIllegalStateWithVerbose() {
      * without {@link CommonArgParser#CMD_VERBOSE} flag.
      */
     @Test
+    @WithSystemProperty(key = IGNITE_CLUSTER_NAME, value = "TEST_CLUSTER_NAME")
```
Comment (vmalin) [inline modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java:1677]: This particular test checks very specific case so I don't think it's a good idea to add extra checks here. Better to choose 3-5 most popular commands and add your checks to some common cases. Also best way to check control script output is to use testOut.toString().contains(%your_pattern%)

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheViewer.java [hunk 6]
```diff
@@ -416,7 +421,8 @@ private void cachesConfig(
      * @param infos Caches info.
      * @param cmd Command.
      */
-    private void printCacheInfos(Collection<CacheInfo> infos, VisorViewCacheCmd cmd, Logger logger) {
+    private void printCacheInfos(Collection<CacheInfo> infos, VisorViewCacheCmd cmd, Logger logger, String clusterName) {
+        logger.info(clusterName);
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheViewer.java:425]: blank line after logger.info

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/Command.java [hunk 7]
```diff
@@ -51,6 +53,29 @@ public static GridClient startClient(GridClientConfiguration clientCfg) throws E
         return client;
     }
 
+    /**
+     * @return Cluster information to show user for.
+     */
+    default String getFullClusterInfo(GridClientClusterState clientCfg){
+        String clusterName = getClusterInfo(clientCfg);
+        String clusterInfo = "Cluster";
+        if (clusterName != null)
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/Command.java:62]: Better to use if-else logic here. If clusterName == null do A, else do B.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/ClusterChangeTagCommand.java (hunk 1 above)
Comment (VladimirGoncharov-GG) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/ClusterChangeTagCommand.java]: Fixed

**Pull request closed without merging**